src/fsharp/ExtensionTyping.fs

```diff
@@ -45,9 +45,9 @@ module internal ExtensionTyping =
     // Detect the host tooling context
     let toolingCompatibleVersions() = 
         if typeof<obj>.Assembly.GetName().Name = "mscorlib" then 
-            [ "net461"; "net452"; "net451"; "net45"; "netstandard2.0"]
+            [ "net471"; "net47"; "net462"; "net461"; "net452"; "net451"; "net45"; "netstandard2.0"]
```

Comment:
This is unrelated but I noted it needed doing - it updates the set of acceptable prefixes for TP components and the order in which they are searched